python/src/diffusionkit/mlx/test-conversion-mlx.py

```diff
@@ -0,0 +1,123 @@
+import mlx.core as mx
```

Comment:
Could you please follow the `UnitTest.TestCase` usage and make this a simple unit test like [this](https://github.com/argmaxinc/DiffusionKit/blob/main/python/src/diffusionkit/tests/mlx/test_diffusion_pipeline.py#L33)? Also, no need to upload to hub from within the test 👍